python/mp/rosary.py

```diff
@@ -272,60 +379,110 @@ def clear_effects(self):
         # thing wonky?
         self.add_effect('set_color', 'all', 0, 0, 0)
 
+
     @dm.expose()
-    def del_effect(self, id):
-        """Delete an active effect by id."""
+    def clear_effects_fade(self):
+        """
+        Just calling clear_effects() is jarring, let's ease it in
+        """
 
-        effect = self.effect(id)
+        for eff in self.effects:
+            eff.fade_out(30)
```

Comment:
Actually, a trigger may leave the effects in place and modify they way they are running by, for example, twisting knobs in a pre-programmed way. In the case of Kelly's pong, the trigger will be used as input to the effect (I think. \me waves hands in front of white board in a an authoritative fashion).